db/migrate/20121028211448_testing.rb

```diff
@@ -13,30 +13,30 @@ def self.up
     end
 
     create_table :test_support_files do |t|
-      t.string :file_name, { :null => false }
-      t.references :assignment, { :null => false }
-      t.text :description, { :null => false }
+      t.string :file_name, **{ :null => false }
```

Comment:
as above, I think you can do `null: false` without the `**{...}`

Reply:
As mentioned in the meeting, this change breaks the migrations, so I've opted against it.

Reply:
@pretendWhale Please push the exact change you made and I'll take a loot